app/api/v1/sick_leaves.rb

```diff
@@ -0,0 +1,47 @@
+module V1
+  class SickLeaves < API
+    helpers do
+      def sick_leave
+        SickLeave.find(params[:id])
+      end
+    end
+
+    resources :sick_leaves do
+      post do
+        params do
+          requires :destination, type: String
+          requires :started_at, type: String
+          requires :ended_at, type: String
+        end
+        SickLeave.create!(params)
```

Comment:
Wrap params into .declare method, otherwise description of params that you wrote above make no sense